### PR TITLE
Common input files

### PR DESCRIPTION
**Pull request by akeeste**
This PR creates common hydroData and geometry input files for the WEC-Sim Applications repo where possible:
Commonly used files are moved to the new directory ``_Common_Input_Files``

TODO
- [x] RM3
- [x] OSWEC
- [x] Sphere
- [x] See if Multiple_Wave_Headings OSWEC can be consolidated with Passive Yaw OSWEC hydrodata --> yes, they can combined as long as there are multiple wave headings. Confirmed with Sal
- [x] @akeeste update the Multiple_Wave_Headings or Passive_Yaw case to use the other's OSWEC hydrodata
- [x] all tests passing

This will speed up our testing suite as BEMIO needs to be significantly less often.

### CODE REVIEW

**Comment by akeeste** [comment]
A note on the sphere, we can easily consolidate the Controls and Free Decay Applications' sphere geometry because they are identical. However the free decay BEM data is a different size than the mean drift sphere and does not contain the necessary mean drift output. So, for now we'll leave the mean drift BEM data separate. If there is ever a chance to easily rerun the free decay BEM case and add mean drift output, we can do that and consolidate the last sphere case together.

**Comment by akeeste** [comment]
Tests are passing and this is ready for a merge. See [this comment ](https://github.com/WEC-Sim/WEC-Sim_Applications/pull/39#issuecomment-1811185057) on why I suspect the MOST test is failing

File: Multiple_Wave_Headings/wecSimInputFile.m [hunk 1]
```diff
@@ -45,32 +42,31 @@
...
-body(1).geometryFile = 'geometry/flap.stl';  % Geometry File
+body(1) = bodyClass('../_Common_Input_Files/OSWEC/hydroData/oswec.h5');      % Initialize bodyClass for Flap
+body(1).geometryFile = '../_Common_Input_Files/OSWEC/geometry/flap.stl';     % Geometry File
 body(1).mass = 12700;                           % User-Defined mass [kg]
-body(1).inertia = [1.85e6 1.85e6 1.85e6];  % Moment of Inertia [kg-m^2]
+body(1).inertia = [1.85e6 1.85e6 1.85e6];       % Moment of Inertia [kg-m^2]
 body(1).yaw.option=1;                           % Turn passive yaw ON
 body(1).yaw.threshold=0.01;                     % Set passive yaw threshold
 
 % Base NOTE: This test uses unique BEM for the OSWEC
```
Comment (jtgrasb) [inline Multiple_Wave_Headings/wecSimInputFile.m:54]: Remove note

File: Passive_Yaw/PassiveYawOFF/wecSimInputFile.m [hunk 2]
```diff
@@ -2,45 +2,45 @@
...
+% Flap
+body(1) = bodyClass('../../_Common_Input_Files/OSWEC/hydroData/oswec.h5');   % Initialize bodyClass for Flap
+body(1).geometryFile = '../../_Common_Input_Files/OSWEC/geometry/flap.stl';  % Geometry File
 body(1).mass = 12700;                           % User-Defined mass [kg]
-body(1).inertia = [1.85e6 1.85e6 1.85e6];  % Moment of Inertia [kg-m^2]
-body(1).yaw.option=0;                           % Turn passive yaw OFF
+body(1).inertia = [1.85e6 1.85e6 1.85e6];       % Moment of Inertia [kg-m^2]
+body(1).yaw.option = 0;                         % Turn passive yaw OFF
 
 % Base NOTE: This test uses unique BEM for the OSWEC
```
Comment (jtgrasb) [inline Passive_Yaw/PassiveYawOFF/wecSimInputFile.m:29]: Remove note

File: Passive_Yaw/PassiveYawON/wecSimInputFile.m [hunk 3]
```diff
@@ -2,47 +2,47 @@
...
+body(1).geometryFile = '../../_Common_Input_Files/OSWEC/geometry/flap.stl';  % Geometry File
 body(1).mass = 12700;                           % User-Defined mass [kg]
-body(1).inertia = [1.85e6 1.85e6 1.85e6];  % Moment of Inertia [kg-m^2]
-body(1).yaw.option=1;                           % Turn passive yaw ON
-body(1).yaw.threshold=0.01;                     % Set passive yaw threshold
+body(1).inertia = [1.85e6 1.85e6 1.85e6];       % Moment of Inertia [kg-m^2]
+body(1).yaw.option = 1;                         % Turn passive yaw ON
+body(1).yaw.threshold = 0.01;                   % Set passive yaw threshold
 
 % Base NOTE: This test uses unique BEM for the OSWEC
```
Comment (jtgrasb) [inline Passive_Yaw/PassiveYawON/wecSimInputFile.m:30]: Remove note

File: Passive_Yaw/PassiveYawRegression/wecSimInputFile.m [hunk 4]
```diff
@@ -17,34 +17,34 @@
...
+body(1) = bodyClass('../../_Common_Input_Files/OSWEC/hydroData/oswec.h5');   % Initialize bodyClass for Flap
+body(1).geometryFile = '../../_Common_Input_Files/OSWEC/geometry/flap.stl';  % Geometry File
 body(1).mass = 12700;                           % User-Defined mass [kg]
 body(1).inertia = [1.85e6 1.85e6 1.85e6];       % Moment of Inertia [kg-m^2]
-body(1).yaw.option=1;                           % Turn passive yaw ON
-body(1).yaw.threshold=1;                        % Set passive yaw threshold
+body(1).yaw.option = 1;                         % Turn passive yaw ON
+body(1).yaw.threshold = 1;                      % Set passive yaw threshold
 
 % Base NOTE: This test uses unique BEM for the OSWEC
```
Comment (jtgrasb) [inline Passive_Yaw/PassiveYawRegression/wecSimInputFile.m:32]: Remove note

**Comment by jtgrasb** [comment]
@akeeste Just made some comments to remove the notes about the unique OSWEC since I don't think this is being used anymore. Other than that, this looks good and is ready for merge. 
I agree with your assessment of the most error. I think we should increase the error bounds.

**Comment by akeeste** [comment]
Thanks for the quick review @jtgrasb. I'll remove those comments for the OSWEC base as well. When I have time, I'll look at another PR to keep the processed MOST input files on the repo and increase the tolerance on the results

**Pull request merged**